.github/workflows/node-ci.yml

```diff
@@ -0,0 +1,41 @@
+name: Node.js CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  test-build-release:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js 12.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12.x
+      - name: Install npm packages using cache
+        uses: bahmutov/npm-install@v1
```

Comment:
Should this not use `yarn install`?

Reply:
It does. See https://github.com/opentripplanner/otp-react-redux/pull/300/checks?check_run_id=1658566171#step:4:14.